draft-ietf-teep-architecture.md

```diff
@@ -1329,23 +1329,47 @@ may need to be updated.
 ## Compromised TAM
 
 Device TEEs are responsible for validating the supplied TAM certificates
-to determine that the TAM is trustworthy. A compromised TAM can install
-arbitrary TAs to, potentially, many devices, which makes TAMs high value
-targets. An advanced, persistent threat actor could potentially compromise
-a TAM without impacting its certificate or raising concern from the TAM's
-operator. To mitigate this threat, TEEs have several options for 
-detecting and mitigating a compromised TAM:
+to determine that the TAM is trustworthy. A compromised TAM may bring multiple threats 
+and damage to user devices that it can managee and the device owners. 
+In one hand, the device information of many devices that the TAM manages and 
+therefore have in its system may be leaked to a bad actor. On the other hand,
+a compromised TAM can install many TAs to launch somee DoS attack to devices,
+for example, by filling up a device's TEE resources reserved for TAs such that 
+other TAs may not get resource to be installed or properly function. It may
```

Comment:
```suggestion
for example, by filling up a device's TEE resources reserved for TAs such that 
other TAs may not get resources to be installed or properly function. It may
```